src/__tests__/router.test.ts

```diff
@@ -0,0 +1,49 @@
+import {server} from '../server'
+const apiJson = require('../../docs/api.json')
+
+type httpMethod = 
+  "GET"
```

Comment:
Think somethings happening with the linting, cos these should all be single quote. That's the second thing that hasn't been caught. Can you have a quick look at what's going on?